src/Workleap.Extensions.Mongo/Indexing/MongoIndexer.cs

```diff
@@ -130,9 +137,14 @@ private async Task UpdateIndexesInternalAsync(IEnumerable<Type> types, IMongoDat
 
         await IndexDeleter.ProcessAsync(database, expectedIndexes, this._loggerFactory, cancellationToken).ConfigureAwait(false);
     }
+    
+    internal static bool IsDocumentTypesWithExplicitMongoCollectionAttribute(Type typeCandidate)
```

Comment:
Should it be private? Unless we want to test it.

Reply:
There are already pre-existing tests for it